src/components/lists/InvestmentListItem.tsx

```diff
@@ -118,6 +118,11 @@ export default function ({
     );
   };
 
+  const getItemKey = (item) => {
+    const { name, balance } = item;
+    return name + balance;
```

Comment:
Can we guarantee that `name` and `balance` are available?

Regarding `name + balance`, if either of them are undefined we'll get a `NaN` or `undefined` in the key which may lead to duplicate keys. Is there a different way to do this using a more unique identifier?

Reply:
If the item is not available then keyExtractor not call